src/naming.adoc

```diff
@@ -78,7 +78,7 @@ of the P extension.
 
 Standard unprivileged extensions can also be named by using a single "Z" followed by an
 alphanumeric name. The name must end with an alphabetical character or an 
```

Comment:
I think this paragraph is combining two things together. The sentence "The name must end with an alphabetical character or an optional version number" is making version number part of the "name". But, as the "Version Numbers" section says, the version number is encoded _after_ the name. The "name" of extensions must end in alphabetical character independent of whether there is a version number.

I think the "Version Numbering" and "Underscores" sections should be moved further down after the extension naming schemes are described. And all references to versions from the naming sections should be removed.

I'm happy to work on the change.

Reply:
@kersten1 I'll be unavailable for a few days, but I'm happy to roll with whatever changes you and Craig agree upon.  Don't wait for me to merge.